cmd/kubeadm/app/util/error.go

```diff
@@ -35,6 +35,10 @@ const (
 	ValidationExitCode = 3
 )
 
+var ErrInvalidSubcmdMsg = "invalid subcommand"
+
+var ErrExit = fmt.Errorf("exit")
```

Comment:
same here and the rest of the PR

please use errors.New()
errors is the github.com/pkg/errors package

Reply:
done. Thanks @neolit123